src/msg/simple/Accepter.cc

```diff
@@ -280,33 +327,67 @@ void *Accepter::entry()
 
   ldout(msgr->cct,20) << "accepter closing" << dendl;
   // don't close socket, in case we start up again?  blech.
-  if (listen_sd >= 0) {
-    ::close(listen_sd);
-    listen_sd = -1;
-  }
-  ldout(msgr->cct,10) << "accepter stopping" << dendl;
+  // if (listen_sd >= 0) {
```

Comment:
@tchaikov 
I left it there because it was the original code their, and part of the discussion with Sage on the possibility of having a race with the FD number being closed and opened.
Let me know if it needs to go.